gc/base/Heap.cpp

```diff
@@ -544,11 +544,11 @@ MM_Heap::getActualSoftMxSize(MM_EnvironmentBase* env)
 
 		uintptr_t nurserySize = totalHeapSize - tenureSize;
 
-		if (nurserySize <= extensions->softMx) {
+		//if (nurserySize <= extensions->softMx) {
```

Comment:
I think this "if" should be replaced by assertion to be sure we do not set `actualSoftMX` to negative value
Also would you please remove commented lines

Reply:
My apologies, this file shouldn't have been in the change. Since "getActualSoftMxSize()" is still used in openj9, the intention here was to add the new implementation of adjustExpansionWithinSoftMax() here, then remove/replace usage of the old getActualSoftMxSize() from openj9, and then finally delete getActualSoftMxSize() from omr.